static/css/style.css

```diff
@@ -34,6 +34,17 @@ body {
 		padding-right: 20px;
 	}
 }
+.CClogo{
+	height: 40px;
+	width: 40px;
+	position: relative;
```

Comment:
not sure exactly as I have a poor understanding of this concept, but is seems right in relation to other div position....